builtin/commit.c

```diff
@@ -1381,6 +1388,19 @@ int cmd_status(int argc, const char **argv, const char *prefix)
 
 	s.is_initial = get_sha1(s.reference, sha1) ? 1 : 0;
 	s.ignore_submodule_arg = ignore_submodule_arg;
+	s.verbose = verbose;
+	if (s.verbose > 1 && status_format == STATUS_FORMAT_PORCELAIN) {
+		/* Capture extra data for very verbose porcelain output. */
+		s.vvp.enabled = 1;
+		if (!s.is_initial)
+			hashcpy(s.vvp.sha_commit, sha1);
```

Comment:
It is safe to leave this unassigned otherwise because `s` is initialized to all-zero by `wt_status_prepare()` (which is called by `status_init_config()` [above](https://github.com/git-for-windows/git/pull/812/commits/b43d3bf57559a1b3de734585129083c1eed7f7a8#diff-8bdf812e0d5d9983d4c8fab34e829994L1362)), so all is fine.